pkg/flags/flags.go

```diff
@@ -312,55 +315,51 @@ https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-g
-		PublishStatusAddress:        *publishStatusAddress,
-		UpdateStatusOnShutdown:      *updateStatusOnShutdown,
-		ShutdownGracePeriod:         *shutdownGracePeriod,
-		PostShutdownGracePeriod:     *postShutdownGracePeriod,
-		UseNodeInternalIP:           *useNodeInternalIP,
-		SyncRateLimit:               *syncRateLimit,
-		HealthCheckHost:             *healthzHost,
-		DynamicConfigurationRetries: *dynamicConfigurationRetries,
-		ListenPorts: &ngx_config.ListenPorts{
-			Default:  *defServerPort,
-			Health:   *healthzPort,
-			HTTP:     *httpPort,
-			HTTPS:    *httpsPort,
-			SSLProxy: *sslProxyPort,
-		},
-		IngressClassConfiguration: &ingressclass.IngressClassConfiguration{
-			Controller:         *ingressClassController,
-			AnnotationValue:    *ingressClassAnnotation,
-			WatchWithoutClass:  *watchWithoutClass,
-			IngressClassByName: *ingressClassByName,
-		},
-		DisableCatchAll:           *disableCatchAll,
-		ValidationWebhook:         *validationWebhook,
-		ValidationWebhookCertPath: *validationWebhookCert,
-		ValidationWebhookKeyPath:  *validationWebhookKey,
-		InternalLoggerAddress:     *internalLoggerAddress,
+		APIServerHost:                *apiserverHost,
+		RootCAFile:                   "",
+		KubeConfigFile:               *kubeConfigFile,
+		Client:                       nil,
+		ResyncPeriod:                 *resyncPeriod,
+		ConfigMapName:                *configMap,
+		DefaultService:               *defaultSvc,
+		Namespace:                    *watchNamespace,
+		WatchNamespaceSelector:       namespaceSelector,
+		TCPConfigMapName:             *tcpConfigMapName,
+		UDPConfigMapName:             *udpConfigMapName,
+		DefaultSSLCertificate:        *defSSLCertificate,
+		PublishService:               *publishSvc,
+		PublishStatusAddress:         *publishStatusAddress,
+		UpdateStatus:                 *updateStatus,
+		UseNodeInternalIP:            *useNodeInternalIP,
+		ElectionID:                   *electionID,
+		UpdateStatusOnShutdown:       *updateStatusOnShutdown,
+		HealthCheckHost:              *healthzHost,
+		ListenPorts:                  &ngx_config.ListenPorts{Default: *defServerPort, Health: *healthzPort, HTTP: *httpPort, HTTPS: *httpsPort, SSLProxy: *sslProxyPort},
+		DisableServiceExternalName:   *disableServiceExternalName,
+		EnableSSLPassthrough:         *enableSSLPassthrough,
+		EnableProfiling:              *profiling,
+		EnableMetrics:                *enableMetrics,
+		MetricsPerHost:               *metricsPerHost,
+		MetricsBuckets:               histogramBuckets,
+		ReportStatusClasses:          *reportStatusClasses,
+		FakeCertificate:              &ingress.SSLCert{},
+		SyncRateLimit:                *syncRateLimit,
+		DisableCatchAll:              *disableCatchAll,
+		IngressClassConfiguration:    &ingressclass.IngressClassConfiguration{Controller: *ingressClassController, AnnotationValue: *ingressClassAnnotation, WatchWithoutClass: *watchWithoutClass, IngressClassByName: *ingressClassByName},
```

Comment:
I think also maybe revert the formatting of this as it is a lot harder to read in a single line